docs/user/multi-user-dfuse.md

```diff
@@ -0,0 +1,114 @@
+# Multi user dfuse.
+
+## Overview
+
+Multi user dfuse is a operating mode of dfuse where it is able to provide filesystem access to
+multiple users/groups on a node through the same dfuse daemon.  This can be used persistently
```

Comment:
```suggestion
Multi user dfuse is an operating mode of dfuse where it is able to provide filesystem access to
multiple users/groups on a node through the same dfuse daemon.  This can be used persistently
```